doc/lua_api.txt

```diff
@@ -3637,6 +3638,15 @@ Definition tables
             end
         ^ The user may be any ObjectRef or nil.
         ]]
+        _custom_field = whatever,
+        --[[
+        ^ Add your own custom fields with your own chosen name to store
+          arbitrary additional information about an item which you can later
+          read them from `minetest.registered_items`. These fields will never
+          be used by the engine but they may be used by mods. By convention,
+          all custom field names should start with `_` to avoid naming
+          collisions with future engine usage.
```

Comment:
All that's needed here is
"Add your own custom fields. By convention, all custom field names should start with `_` to avoid naming collisions with future engine usage."
the rest of your text is obvious and covered by the above.